templates/navbar.html

```diff
@@ -9,7 +9,9 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{{ url_for('index.index') }}">AcousticBrainz</a>
+      <a class="navbar-brand logo" href="{{ url_for('index.index') }}">
+        <img src="{{ url_for('static', filename='img/navbar_logo.svg') }}" alt="AcousticBrainz" />
```

Comment:
Not sure if fallback to PNG is needed there. It looks like SVG is supported pretty much everywhere: http://caniuse.com/#feat=svg.